doc/design.md

```diff
@@ -125,7 +125,7 @@ spec:
 We hope Fluid users could represent the above YAML file by the following line.
 
 ```python
-skaffold_image_leeroy_web = fluid.Image(
+skaffold_image_leeroy_web = fluid.image_resource(
```

Comment:
Similarly, "image" is one of a few pre-defined resource types. How about we keep the name `Image`.